src/js/Controllers/NavController.js

```diff
@@ -0,0 +1,24 @@
+import RpcFactory from './RpcFactory'
+
+class NavController {
+    constructor () {
+        this.addListener = this.addListener.bind(this)
+    }
+    addListener(listener) {
+        this.listener = listener
+    }
+    handleRPC(rpc) {
+        let methodName = rpc.method.split(".")[1]
+        switch (methodName) {
+            case "StartStream":
+                var message = "This system does not support video streaming."
+                return {"rpc": RpcFactory.UnsupportedResourceResponse(rpc, message)};
+            case "SetVideoConfig":
+                var message = "This system does not support video streaming."
+                return {"rpc": RpcFactory.UnsupportedResourceResponse(rpc, message)};
```

Comment:
Shouldn't there be a default case? Otherwise, the HMI won't send any response to other Navigation RPCs.

Reply:
Should probably directly respond to `IsReady` with `available=false` as well

Reply:
Should also respond to Audio Streaming RPCs.